Runtime/MeshOperations/CombineMeshes.cs

```diff
@@ -136,56 +136,16 @@ public static List<ProBuilderMesh> Combine(ProBuilderMesh meshTarget, IEnumerabl
             if (!meshes.Any() || meshes.Count() < 2 )
                 return null;
 
-            var vertices = new List<Vertex>();
-            var faces = new List<Face>();
-            var autoUvFaces = new List<Face>();
-            var sharedVertices = new List<SharedVertex>();
-            var sharedTextures = new List<SharedVertex>();
-            int offset = 0;
-            var materialMap = new List<Material>();
-            var targetTransform = meshTarget.transform;
-
-
-            //First pass put the meshTarget as first vertices
-            foreach (var mesh in meshes)
-            {
-                if (mesh == meshTarget)
-                {
-                    var meshVertexCount = mesh.vertexCount;
-                    var meshVertices = mesh.GetVertices();
-                    var meshFaces = mesh.facesInternal;
-                    var meshSharedVertices = mesh.sharedVertices;
-                    var meshSharedTextures = mesh.sharedTextures;
-                    var materials = mesh.renderer.sharedMaterials;
-                    var materialCount = materials.Length;
-
-                    for (int i = 0; i < meshVertexCount; i++)
-                        vertices.Add(meshVertices[i]);
-
-                    foreach (var face in meshFaces)
-                    {
-                        faces.Add(face);
-                    }
-
-                    foreach (var sv in meshSharedVertices)
-                    {                   
-                        sharedVertices.Add(sv);
-                    }
-
-                    foreach (var st in meshSharedTextures)
-                    {
-                        sharedTextures.Add(st);
-                    }
-
-                    foreach (var mat in materials)
-                    {
-                        materialMap.Add(mat);
-                    }
+            if (!meshes.Contains(meshTarget))
```

Comment:
@karl-, @MartinImgSpc:
- Am I right to see this function as this following?
> Here a list of meshes to combine, please use this one as the host/starting point/mesh of reference/target

If so, it would make sense to enforce this condition. Also, I would change its signature to help better understand what it does and to keep a consistent API:
```C#
List<ProBuilderMesh> Combine(IEnumerable<ProBuilderMesh> meshes, ProBuilderMesh meshTarget)
```

- @karl- , is it fine to add a new API function?

Reply:
Yes, ok to add a new func signature with the `ProBuilderMesh` as an overload.